src/shaders/symbol_sdf.fragment.glsl

```diff
@@ -1,12 +1,19 @@
+#define SDF_PX 8.0
+#define BLUR_OFFSET 1.19
+#define HALO_OFFSET 6.0
+#define GAMMA 0.105/DEVICE_PIXEL_RATIO
```

Comment:
LOL, I totally agree with you that these names are cryptic.  And here I must confess, I just did a lazy/dumb transcription from `draw_symbol.js` (where these constants / calculations lived before this), without actually discerning what they meant.  Lemme see if I can decipher them into more meaningful names.